lib/cqm/converter/qdm_patient.rb

```diff
@@ -156,6 +160,15 @@ def extractor(qdm_thing)
       end
     end
 
+    def location_extractor(fac)
+      inter = interval_extractor(fac[:locationPeriod].as_json.symbolize_keys)
```

Comment:
Is `interval` a reserved word? I definitely always prefer spelling these variable names out.

Reply:
Same with `fac`.

Reply:
Yeah, I'll fix that.